LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
Okay this is more a matter of religion I guess but I am fine with MIT for our wrappers.  I am not sure whether we must require the same license for all of our wrappers?